tests/EventStore/EventStoreTest.php

```diff
@@ -96,7 +95,7 @@ protected function createDomainMessage($id, $version, $recordedOn = null)
         return new DomainMessage(
             $id,
             $version,
-            new SomethingHappened(),
+            new SomethingHappened($recordedOn),
             $recordedOn ? $recordedOn : new \DateTimeImmutable("now")
```

Comment:
Why not use the new php 7 null coalesce operator here? $recordedOn ?? new \DateTimeImmutable("now")